microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
```

Comment:
I think this file may have been intended as a header, but is being rendered as a sub-component.

Reply:
I think it should be a subcomponent under "Configuring persistent storage for MicroShift," right? 

Or should it be its own header? If so, what should I do with "Configuring persistent storage for MicroShift"? Should it be there at all?